nupic/frameworks/opf/model.py

```diff
@@ -24,12 +24,16 @@
 import cPickle as pickle
 import os
 import shutil
+import logging
 from abc import ABCMeta, abstractmethod
 
 import nupic.frameworks.opf.opfutils as opfutils
 
-
 ###############################################################
+# global variable
+global globalModelsStorage
```

Comment:
This is the wrong context for usage of `global` keyword.  It need only be used inside functions.